SpotifyAPI/Local/SpotifyLocalAPI.cs

```diff
@@ -149,21 +157,72 @@ public StatusResponse GetStatus()
         }
 
         /// <summary>
-        /// Mutes Spotify (Requires nircmd.dll)
+        /// Mutes Spotify (Requires Windows 7 or newer)
         /// </summary>
         public void Mute()
         {
-            if (File.Exists("nircmd.dll"))
-                DoNirCmd("muteappvolume spotify.exe 1");
+            //Windows < Windows Vista Check
+            Contract.Requires(Environment.OSVersion.Version.Major >= 6);
```

Comment:
The main reason for using Contracts was the option to immediately throw an Exception if an argument is invalid. I became accustomed to it, in hindsight I should have stayed with if-throw-else. The main advantage for continuous Contract usage is the possibility to combine it with IntelliSense
![figure 2](https://cloud.githubusercontent.com/assets/6277666/10702065/8fd9dc28-79c7-11e5-8bca-e374a7951577.png)